src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -68,14 +68,6 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       SkipGetTargetFrameworkProperties="true"
       UndefineProperties="TargetFramework;TargetFrameworks;RuntimeIdentifier;PublishDir" />
 
-    <!-- Omit System.Security.Permissions closure. https://github.com/dotnet/runtime/issues/54341
-         It is brought in by System.Security.Cryptography.Xml for netstandard2.0 compatibility
-         but not required in net6.0 since the types required exist in System.Security.AccessControl. -->
-    <Reference Include="Microsoft.Win32.SystemEvents" ExcludeAssets="All" />
-    <Reference Include="System.Drawing.Common" ExcludeAssets="All" />
-    <Reference Include="System.Security.Permissions" ExcludeAssets="All" />
-    <Reference Include="System.Windows.Extensions" ExcludeAssets="All" />
-
```

Comment:
I wonder if you can remove the stuff in `_DisallowedReferenceAssemblies` now?

Reply:
Hmm, _maybe_❔ I'll look at a binlog once the CI wraps up.